Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -142,59 +142,60 @@ def http_request(uri: str, method: str, headers: Dict = {},
             'Request Failed. with status: ' + str(result.status_code) + '. Reason is: ' + str(result.reason))
 
     # if pcap download
-    if params.get('type') == 'export':
+    if is_pcap:
         return result
 
     json_result = json.loads(xml2json(result.text))
 
     # handle non success
-    if json_result['response']['@status'] != 'success':
-        if 'msg' in json_result['response'] and 'line' in json_result['response']['msg']:
-            # catch non existing object error and display a meaningful message
-            if json_result['response']['msg']['line'] == 'No such node':
+    if 'response' in json_result and '@code' in json_result['response']:
```

Comment:
this is the exact same condition from [line 201](https://github.com/demisto/content/pull/9148/files#diff-bc4875303117322206b85fe1ffb69cddR201)
and the is nothing in between those blocks of code.

also since the rest of the function logic lays within the condition, it is better to reverse the condition and minimize the nesting, see [point number 1](https://www.tutorialdocs.com/article/python-conditional-statements-tips.html).